code/controllers/subsystem/jobs.dm

```diff
@@ -238,6 +238,8 @@ var/datum/subsystem/job/SSjob
 	for(var/mob/new_player/player in unassigned)
 		if(jobban_isbanned(player, "catban"))
 			AssignRole(player, "Assistant")
+		if(jobban_isbanned(player, "cluwneban"))
```

Comment:
do tell how, previous attempts at doing so failed miserably and the entire code team is filled with dweabs.